packages/client/tests/functional/tracing/tests.ts

```diff
@@ -20,17 +21,42 @@ type Tree = {
 }
 
 function buildTree(tree: Tree, spans: ReadableSpan[]): Tree {
-  // @ts-ignore - For JSON stringify debugging
-  delete tree.span._spanProcessor
-
   const childrenSpans = spans.filter((span) => span.parentSpanId === tree.span.spanContext().spanId)
   if (childrenSpans.length) {
     tree.children = childrenSpans.map((span) => buildTree({ span }, spans))
   } else {
     tree.children = []
   }
 
-  return tree
+  // Remove unused keys for easier debugging
+  const simpleTree = JSON.stringify(
```

Comment:
Is it still needed?

Reply:
Its handy to keep so that when someone console logs the tree it's really small - I will keep it.